### PR TITLE
GSAGH-497: GsaGH creates releases opposite to what is defined using 'Edit element/member' component

### PR DESCRIPTION
**Pull request by SandeepArup**

### CODE REVIEW

File: GsaGH/Components/2_Geometry/Edit1dMember.cs [hunk 1]
```diff
@@ -268,12 +268,12 @@ protected override void SolveInstance(IGH_DataAccess da) {
...
       GsaBool6Goo start = null;
       if (da.GetData(8, ref start)) {
-        mem.ReleaseStart = start.Value;
+        mem.ReleaseStart = start.Value.Negate();
       }
 
       GsaBool6Goo end = null;
       if (da.GetData(9, ref end)) {
-        mem.ReleaseEnd = end.Value;
+        mem.ReleaseEnd = start.Value.Negate();
```
Comment (psarras) [inline GsaGH/Components/2_Geometry/Edit1dMember.cs:276]: you mean `end`

Comment (psarras) [reply GsaGH/Components/2_Geometry/Edit1dMember.cs:276]: a bit confusing, we negate the input, so that we can negate it again while we set the values?

**Review by psarras — CHANGES_REQUESTED** [review]
We would need to think a bit more about it. It feels like it would be more obvious if we create a RestrainBool and a ReleaseBool Parameters, instead of having a flag.

File: GsaGH/Components/2_Geometry/Edit1dMember.cs (hunk 1 above)
Comment (SandeepArup) [reply GsaGH/Components/2_Geometry/Edit1dMember.cs]: We negate input to pass correct value to Gsa Member. We negate output to display information correctly.

**Pull request closed without merging**